### DISCUSSION TITLE
Changing BetterFeedback localization after initialization

### DISCUSSION THREAD

**Discussion by david640** [ueman/feedback#331] [Q&A]
Thank you for the brilliant library. In my app users can change localization at any time. So when they want to submit feedback I want to change the BetterFeedback localization to the current locale that the user has selected e.g. pl_PL. I understand that I can use localeOverride on initialization in main(), which is fine, but this does not help if a user changes locale after initialization. Does anyone know if this is possible and if so how to do it ?

**Comment by ueman** [comment]
It heavily depends on how you've built your language switcher functionality. Without an example, I can't really provide guidance here

**Reply by david640** [reply]
I am using flutter_localizations/flutter_localizations.dart package. (l10n) When a user selects a new locale from a drop down list I set the new locale by using S.load(Locale(e.g. pl_PL)) where S is the generated l10n class.

**Reply by ueman** [reply]
You should be able to essentially use the same approach for this library. Just pass localization delegate that's backed by your localizations, and then it should just work.

**Reply by david640** [reply]
Thanks for your reply, but I do not understand what you mean by "_Just pass localization delegate that's backed by your localizations_". I would be grateful if you could clarify where do I need to pass it ?
